examples/vision/supervised-contrastive-learning.py

```diff
@@ -0,0 +1,186 @@
+"""
+Title: Supervised Contrastive Learning
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2020/11/01
+Last modified: 2020/11/01
+Description: Using supervised contrastive learning for image classification.
+"""
+
+## Introduction
+
+[Supervised Contrastive Learning](https://arxiv.org/abs/2004.11362) (Prannay Khosla et al.) is a training methodology that outperforms cross-entropy on supervised learning tasks. 
+
+Essentially, training an image classification model with Supervised Contrastive Learning is peformed in two phases: 
+
+1. Pre-training an encoder to generate feature vectors for input images such that feature vectors of images in the same class will be more similar compared feature vectors of images in other classes.
+2. Training a classifier on top of the freezed encoder.
+
+## Setup
+"""
+
+!pip install tensorflow-addons
+
+import tensorflow as tf
+import tensorflow_addons as tfa
+import numpy as np
+import keras
+from keras import layers
+from keras.losses import Loss
```

Comment:
Just use `keras` for this, don't import `Loss`

Reply:
Done